index.html

```diff
@@ -5,19 +5,65 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one,two,three">
 
-		<title>Blank Template</title>
+		<title>Relaxr</title>
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,600,800' rel='stylesheet' type='text/css'>
 
 		<!-- CSS links -->
 
+		<link rel="stylesheet" type="text/css" href="css/reset.css">
+
+		<link rel="stylesheet" type="text/css" href="css/style.css">
+		
 	</head>
 	<body>
 		<div>
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<h1 class="h1">Relaxr</h1>
+				<h2 class="h2">Get piece of mind with a single tap</h2>
+				<div class="center">
+					<button class="button1">Get it Now</button>
+				</div>
+				<div class="paddinglarge"></div>
 			</header>
-			<footer>
-				&copy; Footer Goes Here
+			<main class="clear">
+				<div class="benefitsbg">
+					<div class="benefitswidth">
+						<h3 class="h3">Benefits</h3>
+						<p class="pwidth">The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+						<ul class="list">
+							<li>Schedule meetings for you</li>
+							<li>Excel automation to complete your work for you</li>
+							<li>Responds to emails on your behalf</li>
+							<li>Does all your work for you with our revolutionary AI technology</li>
+						</ul>
+					</div>
+				</div>
+				<div>
+					<blockquote class="blockquote">
+						<p>
```

Comment:
Really great job with the readability of your code - nested elements and structural HTML.